cmd/sonobuoy/app/images.go

```diff
@@ -91,6 +92,39 @@ func NewCmdImages() *cobra.Command {
 	return cmd
 }
 
+func listCmd() *cobra.Command {
+	var flags imagesFlags
+
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List images",
+		Run: func(cmd *cobra.Command, args []string) {
+			var client image.Client
+			if flags.dryRun {
+				client = image.DryRunClient{}
+			} else {
+				client = image.NewDockerClient()
+			}
+			version, err := getClusterVersion(flags.k8sVersion, flags.kubeconfig)
+			if err != nil {
+				errlog.LogError(err)
+				os.Exit(1)
+			}
+			if err := listImages(flags.plugins, version, client); err != nil {
+				errlog.LogError(err)
+				os.Exit(1)
+			}
+		},
+		Args: cobra.ExactArgs(0),
+	}
+	AddKubeconfigFlag(&flags.kubeconfig, listCmd.Flags())
+	AddPluginListFlag(&flags.plugins, listCmd.Flags())
+	AddDryRunFlag(&flags.dryRun, listCmd.Flags())
```

Comment:
Any reason not to add the dryRun flag to the `images` command as well? Apparently we thought it had a dryrun option since the logic references but maybe that was just boilerplate.

Reply:
Done.